### PR TITLE
Faq storyblok component

### PR DESCRIPTION
**Pull request by annarhughes**
Adds an Faq storyblok component, to be used within a storyblok `page_section` component (`StoryblokPageSection.tsx` in our app)

### CODE REVIEW

File: components/common/Faqs.tsx [hunk 1]
```diff
@@ -51,7 +51,11 @@ const Faqs = (props: FaqsProps) => {
               {t.rich(faq.body, {
                 partnerName: partnerName,
                 ...(faq.link && {
-                  faqLink: (children) => <Link href={faq.link ? faq.link : '#'}>{children}</Link>,
+                  faqLink: (children) => (
+                    <Link href={faq.link ? faq.link : '#'} target="_blank">
```
Comment (swetha-chayn) [inline components/common/Faqs.tsx:55]: It's so annoying that it won't figure out that `faq.link` is defined!

**Review by swetha-chayn — APPROVED** [review]
🌟 lgtm

File: components/storyblok/StoryblokFaqs.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+const StoryblokFaqs = (props: StoryblokFaqsProps) => {
+  const { faqs } = props;
+
+  const handleChange = (faqTitle: string) => (event: React.SyntheticEvent, isExpanded: boolean) => {
+    if (isExpanded) {
+      logEvent(FAQ_OPENED, { faqTitle: faqTitle });
+    }
+  };
```
[the remark this answers is not in the document]
Comment (swetha-chayn) [reply components/storyblok/StoryblokFaqs.tsx:28]: (line 20)

[the remark this answers is not in the document]
Comment (annarhughes) [reply components/storyblok/StoryblokFaqs.tsx]: done :)

File: components/common/Faqs.tsx (hunk 1 above)
Comment (annarhughes) [reply components/common/Faqs.tsx:55]: yeah theres a few places this happens and more code than you'd want :(

**Review by swetha-chayn — APPROVED** [review]
woo!

**Pull request merged**